cmd/cli/node/columns.go

```diff
@@ -24,6 +24,10 @@ var alwaysColumns = []output.TableColumn[*models.NodeInfo]{
 		ColumnConfig: table.ColumnConfig{Name: "type"},
 		Value:        func(ni *models.NodeInfo) string { return ni.NodeType.String() },
 	},
+	{
+		ColumnConfig: table.ColumnConfig{Name: "approval"},
```

Comment:
LGTM, but "approval" feels like the wrong word.

"status"? -> "waiting approval", "in-cluster", etc.

I dunno - this feels VERY useful, but is it worth being present in the table when 99.9% of the lifetime of the node it'll be in one state?

Maybe just an emoji for status? "⏳", "❌" or "⛔️", "✅" ? 

just brainstorming

Reply:
yeah, i'm generally with you - the reason i like emojis is that they're fun, they're localized already, they're super compact (just one character), etc